eng/pipelines/templates/stages/platform-matrix-excluding-pypy.json

```diff
@@ -6,11 +6,11 @@
   },
   "matrix": {
     "Agent": {
-      "ubuntu-20.04": { "OSVmImage": "MMSUbuntu20.04", "Pool": "azsdk-pool-mms-ubuntu-2004-general" },
       "windows-2022": { "OSVmImage": "MMS2022", "Pool": "azsdk-pool-mms-win-2022-general" },
+      "ubuntu-20.04": { "OSVmImage": "MMSUbuntu20.04", "Pool": "azsdk-pool-mms-ubuntu-2004-general" },
```

Comment:
We may want to restore the original order of these to avoid 3.11 on Windows (per last CI run)

Reply:
Yep. I have done so.